fusestore/store/src/meta_service/state_machine.rs

```diff
@@ -409,8 +409,9 @@ impl StateMachine {
     }
 
     pub fn get_data_parts(&self, db_name: &str, table_name: &str) -> Option<Vec<DataPartInfo>> {
-        let parts = self.tbl_parts.get(db_name);
-        parts.and_then(|m| m.get(table_name)).map(Clone::clone)
+        let db = self.databases.get(db_name).unwrap();
```

Comment:
`unwrap` may not be a good idea, I guess you just forget to clean up it (them) :D